src/validation.cpp

```diff
@@ -713,6 +714,8 @@ bool VerifyAmounts(const CCoinsViewCache& cache, const CTransaction& tx, std::ve
     memset(explBlinds, 0, sizeof(explBlinds));
     int ret;
 
+    uint256 wtxid(tx.GetHashWithWitness());
```

Comment:
Why not move this closer to where it's used? Or perhaps even inline it in its only use?

Reply:
used to use it in this commit, ill move to other commit